src/components/Room/Room.js

```diff
@@ -48,6 +60,317 @@ function Room() {
     }
   },[userInRoom]);
 
+  /**
+   * @function toggleAudio - toggles mute on user's audio.
+   */
+  function toggleAudio() {
+    let toggleAudioData = {
+      'action': 'ToggleAudioStream',
+    }
+    dispatchMediaStreams(toggleAudioData)
+  }
+
+  /**
+   * @function toggleVideo - toggles mute on user's video.
+   */
+  function toggleVideo() {
+    let toggleVideoData = {
+      'action': 'ToggleVideoStream',
+    }
+    dispatchMediaStreams(toggleVideoData)
+  }
+
+  /**
+   * @function toggleMediaTracks - enables or disables audio or video tracks on a media stream.
+   * @param {HTMLMediaElement} stream - A media stream.
+   * @param {bool} isAudio - If the media stream is of audio type.
+   */
+  function toggleMediaTracks(stream, isAudio) {
+    if (stream === null) {
+      console.log('Invalid video stream, cannot toggle media.')
+      return
+    }
+    let localMediaTracks = stream.getVideoTracks();
+    let mediaType = 'video'
+    if (isAudio) {
+      mediaType = 'audio'
+      localMediaTracks = stream.getAudioTracks();
+    }
+    if (localMediaTracks.length > 0) {
+      // Disable or re-enable tracks on local stream.
+      console.log('Toggling %o tracks from local stream', mediaType)
+      for (var i = 0; i < localMediaTracks.length; i++) {
```

Comment:
change `var` to `let` in all for loops in this file

Reply:
done